src/app/_components/MarkdownContent.tsx

```diff
@@ -1,13 +1,59 @@
 import Image from 'next/image'
 
-import ReactMarkdown from 'react-markdown'
+import * as Sentry from '@sentry/node'
+import ReactMarkdown, { type Components } from 'react-markdown'
 import rehypeRaw from 'rehype-raw'
 import remarkGfm from 'remark-gfm'
 
+import { graphicsData } from '@/data/graphicsData'
+
 import { buildImageSizeProp } from '@/utils/buildImageSizeProp'
 
+import { TextLink } from '@/components/TextLink'
+
 type MarkdownContentProps = {
-  children: string
+  children: Parameters<typeof ReactMarkdown>[0]['children']
+}
+
+const MarkdownImage: Components['img'] = ({ src, alt }) => {
+  const commonProps = {
+    quality: 100,
+    width: 800,
+    height: 450,
```

Comment:
These two values are weird but I'm not sure how to fix them.

`width: 800` means occupy the full width, because the container's max width is `672px`. But we can't pass `width=full`, Next Image expects a numerical value.

How about we create a variable to make this value clearer?

```ts
const MAX_WIDTH_OF_THE_CONTAINER = "672px"
// ...
width: MAX_WIDTH_OF_THE_CONTAINER
```

As for `height: 450`, this value is meaningless because it's not used, but we can't remove it. How about this? 😅

```ts
const PLACEHOLDER_NUMERICAL_VALUE_TO_MAKE_NEXT_IMAGE_HAPPY=1 
// ...
height: PLACEHOLDER_NUMERICAL_VALUE_TO_MAKE_NEXT_IMAGE_HAPPY
```

Reply:
Why is the container's max-width `672px`?

Reply:
This actually makes me think that we should want to pass these values?

Reply:
> Why is the container's max-width 672px? Don't we have different container widths?

It comes from `m-auto max-w-2xl space-y-16` in `src/app/blog/[slug]/page.tsx`. max-w-2xl = 672px

> This actually makes me think that we should want to pass these values?

Pass which values?

Reply:
I'll create a ticket for this, and move this PR forward. Thank you.